spec/app/ics-029-fee-payment/README.md

```diff
@@ -96,7 +99,33 @@ function PayTimeoutFee(packet: Packet, timeout_relayer: string) {
 }
 ```
 
-These functions will then be utilized in the ICS-4 handlers like so:
+
+The fee module should also expose the following queries so that relayers may query their expected fee:
+
+```typescript
+// Gets the fee expected for submitting ReceivePacket msg for this packet
+function GetReceiveFee(packet) Fee
```

Comment:
I would simplify this with a `PacketID`, which would uniquely identify the packet without requiring the entire data.

This could be `(port_id, channel_id, sequence)` which should be unique on this chain.